docs/synthetics/ssl-certs.md

```diff
@@ -0,0 +1,62 @@
+title: SSL Certificate Monitoring
+description: Monitor & get notified on SSL certificate expiry and change. 
+
+Issues with SSL certificates not only cause downtime of APIs and websites but also hurts their reputation. Sematext Synthetics performs the following SSL certificate checks on all the certificates in the chain - the leaf, intermediate, and root certificates.
+
+* Validate the SSL certificates of the API/websites in every run
+* Check for the expiration of the certificates every day and alert you 30, 14, 7 and 3 days before the expiry
+* Check for certificate change every 10 mins and alert you on detecting any changes, with a detailed change report
+
+Sematext Synthetics alerts you on the failure of these checks via the configured monitor alert channels like Email, Slack, Webhooks, etc. Apart from these checks, Synthetics also provides an SSL certificate report, with details of all the certificates in the chain.
+
+Both the HTTP and Browser monitors perform these checks except for Browser monitors with user journey script configured.
+
+## Certificate Validation
+
+For every run of the monitor, Synthetics performs a set of validity checks on SSL certificates sent by the server. Synthetics will mark the monitor status as failing if any of these checks fail. Synthetics will raise a run failure alert with the details of the failure. Self-signed certificates are not supported. Any websites or APIs using self-signed certificates will fail.
+
+The HTTP monitor performs the following checks:
+
+* If there is a valid chain, connecting the leaf certificate of the server to a trusted root
+* Certificate has not expired
+* Certificate is active (if the certificate is not used before its validity starts)
+* Verify name constraints, if any
+* Verify if there are too many intermediate certificates (more than 3 intermediates)
+* Hostname validation, check if the hostname of the website/API is present in the list of allowed DNS names of the certificate
+
+The browser monitor loads the website in a real Google Chrome browser so validates all the checks done by Google Chrome browser. We update the Chrome browser periodically, so the list of checks stays up-to-date with recent implementation. Along with all the above checks done by the HTTP monitor, the browser monitor performs the following extra checks:
+
+* Certificate is revoked.
+* Certificate uses a weak signature algorithm (e.g. SHA-1)
+* Certificate authority is not trusted
+* Certificate validity is too long
+* Certificate uses a weak key ( e.g. a too small RSA key)
+
+## Certificate Expiry
+
+Sematext Synthetics checks the certificate expiry every day and alerts you via the configured monitor alert channels multiple times before the expiry so that you remained about the expiry. The monitor performs the expiry check for all the certificates in the chains - leaf, intermediates, and root certificate. The Monitor Overview page displays the expiry time of the recently expiring certificate.
```

Comment:
Sematext Synthetics checks the certificate expiry every day and alerts you via the configured monitor alert channels multiple times before the expiry so that you remained about the expiry. The monitor performs the expiry check for all the certificates in the chains - leaf, intermediates, and root certificate. The Monitor Overview page displays the expiry time of the recently expiring certificate.

=>

Sematext Synthetics checks the certificate expiry every day and alerts you via the configured monitor alert channels multiple times before it expires. We make sure you're reminded about the expiry multiple times.

The monitor performs the expiry check for all the certificates in the chain - leaf, intermediates, and root certificate. The Monitor Overview page displays the expiry time of the recently expiring certificate.